account-kit/core/src/actions/disconnect.ts

```diff
@@ -46,17 +46,36 @@ export async function disconnect(config: AlchemyAccountsConfig): Promise<void> {
 // Persistence of Wallet Connect connection state on error.
 const clearWalletConnectStore = () => {
   // Open Wallet Connect Indexed DB
+  let walletConnectDBExists = true;
```

Comment:
we should probably start with the assumption that it isn't initialized

Reply:
Actually on further testing, it is better we start with the assumption that it is in fact initialized. This is because `dbRequest.onUpgradeneeded()` isn't actually called on every request, only when there is a DB version change. This means when we call `disconnect()` after walletConnect has created the IDB store, we wouldn't be running that event handler, and thus if we default to false, we wouldn't have a way to change the value to true.